examples/events/EiffelArtifactDeployedEvent/simple.json

```diff
@@ -0,0 +1,24 @@
+{
+  "meta": {
+    "type": "EiffelArtifactDeployedEvent",
+    "version": "1.0.0",
+    "time": 1234567890,
+    "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee4",
+    "source": {
+      "domainId": "example.domain"
+    }
+  },
+
```

Comment:
Nitpicking: please remove empty lines and spaces and break between the brackets, i.e.
```
"data": {
},
```

instead of
```

"data" : {},

```